adapter.js

```diff
@@ -57,6 +57,11 @@ Adapter.prototype.tip = function (callback) {
   this.blockchain.tip(callback)
 }
 
+// returns blockchain chain tip height
+Adapter.prototype.tipHeight = function (callback) {
+  this.blockchain.tipHeight(callback)
```

Comment:
Could you simply use `this.blockchain.tip((err, res) => callback(err, res && res.height))`